maintainers/maintainer-list.nix

```diff
@@ -10768,4 +10768,10 @@
     github = "felixscheinost";
     githubId = 31761492;
   };
+  benneti = {
+    name = "Benedikt Tissot";
+    email = "benedikt.tissot@googlemail.com";
+    github = "benneti";
+    githubId = 11725645;
+  };
```

Comment:
if you could do this in a separate commit that would be great. we usually do this because reverts or backports are better that way (and a bit nicer history).

Reply:
done